src/core_plugins/metrics/public/components/visualization.js

```diff
@@ -63,18 +64,22 @@ function Visualization(props) {
 
   const component = types[model.type];
   if (component) {
-    return React.createElement(component, {
-      dateFormat: props.dateFormat,
-      reversed: props.reversed,
-      backgroundColor: props.backgroundColor,
-      model: props.model,
-      onBrush: props.onBrush,
-      onChange: props.onChange,
-      onUiState: props.onUiState,
-      uiState: props.uiState,
-      visData: visData.type === model.type ? visData : {},
-      getConfig: props.getConfig
-    });
+    return (
+      <I18nProvider>{
```

Comment:
Not sure that we should use I18nProvider here. Instead try to find place where ReactDOM.render is used and use I18nProvider there